doc/HOWTO/documenting-macros.md

```diff
@@ -0,0 +1,53 @@
+Documenting Macros
+==================
+
+"make doc-nits" (or the check-docs Action) might tell you something like:
+
+~~~~text
```

Comment:
I wasn't aware until today that you can also use [tilde characters instead of backticks](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks) for fencing code blocks. However, since backticks are advertised way to do it in GitHub Markdown and since we use them everywhere else, I really would prefer if you would not introduce this new syntax.

Reply:
It also does not make sense to add the `text` language specifier, since it has no effect. Just three ` ``` `, nothing else. Four backticks should only used to fence a text which displays triple backticks, like this:
````
```
BIO_get_dgram_dest(3) is supposedly internal
(maybe missing from other.syms) but is documented as public
```
````
And BTW: would you please be so kind and fix this location as well? Thanks!

Reply:
```
./doc/HOWTO/documenting-macros.md:6: MD040 Fenced code blocks should have a language specified
./doc/HOWTO/documenting-macros.md:22: MD040 Fenced code blocks should have a language specified
./doc/HOWTO/documenting-macros.md:32: MD040 Fenced code blocks should have a language specified
./doc/HOWTO/documenting-macros.md:43: MD040 Fenced code blocks should have a language specified
```
So I will put "text" back in.